src/iv/ivgl.h

```diff
@@ -96,6 +96,8 @@ class IvGL : public QOpenGLWidget, protected QOpenGLExtraFunctions {
                             GLenum& gltype, GLenum& glformat,
                             GLenum& glinternal) const;
 
+    void clamp_view_to_window();
```

Comment:
This is the only change I Am not confident about. In order to use `clamp_view_to_window` I had to move it from protected to public. I don't know if this is a big deal. I seek some guidance here.